coldfront/core/utils/mail.py

```diff
@@ -125,7 +125,7 @@ def send_allocation_customer_email(allocation_obj, subject, template_name, url_p
     email_receiver_list = []
     for allocation_user in allocation_users:
         if allocation_user.allocation.project.projectuser_set.get(
-                                user=allocation_user.user).enable_notifications:
+                                user=allocation_user.user):
```

Comment:
Why was this removed? I believe we'll want to keep this?

Reply:
Re-added. This was an oversight. 

However, it is worth noting that EULA emails should maybe follow a different opt-in/opt-out model than general notifications. It may be worth adding, for instance a `ALLOW_ALLOCATION_EULA_EMAIL_OPT_OUT`  option and associated logic.